docs/containers.conf.5.md

```diff
@@ -164,6 +164,11 @@ Options are:
   `private` Create private IPC Namespace for the container.
   `host`    Share host IPC Namespace with the container.
 
+**keyring**=true
+
+Indicates whether the container engines create a kernel keyring for use within
+the container.
+
 **label**=true
 
 Indicates whether the container engines use MAC(SELinux) container separation via via labeling. Flag is ignored on disabled systems.
```

Comment:
old
```suggestion
Indicates whether the container engine uses MAC(SELinux) container separation via labeling. This option is ignored on disabled systems.
```

Reply:
Should this be "disabled systems" or "label disabled systems" like shown below https://github.com/containers/common/pull/354/files#diff-f38944fdd6507cc37badbde5c66785bdebb77bd6fa2ea0674cd8117f89e95676R155